dask_glm/algorithms.py

```diff
@@ -134,12 +135,28 @@ def newton(X, y, max_iter=50, tol=1e-8, family=Logistic):
     return beta
 
 
-def admm(X, y, lamduh=0.1, rho=1, over_relax=1,
+def admm(X, y, reg=L1, lamduh=0.1, rho=1, over_relax=1,
```

Comment:
Nope, it works with *any* regularizer which is one reason it's awesome!  If you look at pg. 65 of the [original paper](http://web.stanford.edu/~boyd/papers/pdf/admm_distr_stats.pdf), the `z` update is just computing the "proximal operator" for the regularizer.  In the case of `l1`, that's shrinkage.